src/components/ComponentPages/TopicDetails/CampInfoBar/index.tsx

```diff
@@ -60,37 +60,38 @@ const CampInfoBar = ({ payload = null, isTopicPage = false }) => {
     if (!isTopicPage && payload && Object.keys(payload).length > 0) {
       getBreadCrumbApiCall();
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [payload]);
 
-  useEffect(() => {
-    async function getTreeApiCall() {
-      setLoadingIndicator(true);
-      const reqBody = {
-        topic_num: +router?.query?.camp?.at(0)?.split("-")?.at(0),
-        camp_num: +router?.query?.camp?.at(1)?.split("-")?.at(0),
-        as_of: asof,
-        asofdate:
-          asof == ("default" || asof == "review")
-            ? Date.now() / 1000
-            : asofdate,
-        algorithm: algorithm,
-        update_all: 1,
-      };
-      await Promise.all([
-        getCurrentTopicRecordApi(reqBody),
-        getCurrentCampRecordApi(reqBody),
-      ]);
-      setLoadingIndicator(false);
-    }
-    if (isTopicPage) {
-      getTreeApiCall();
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [
-    asofdate,
-    algorithm,
-    isTopicPage ? +router?.query?.camp[1]?.split("-")[0] : "",
-  ]);
+  // useEffect(() => {
```

Comment:
We can delete this if it is not required.

Reply:
Removed